index.html

```diff
@@ -14,6 +14,21 @@
       <div id="ui">
 	<div id="sidebar">
 	  <div id="controls">
+      <h3>Date Range</h3>
+      <form name="dateSelection">
+        <div class="dateRadio">
+          <input type="radio" name="radioDate" id="radioMonth" checked>
```

Comment:
Ah, you're exactly right. I was testing when we still had inline `<script>` tags and the functions were being hoisted to the top of `index.html`. The way you have it is better practice anyway, so let's keep it.